### DISCUSSION TITLE
Carousel Mouse-Draggable

### DISCUSSION THREAD

**Discussion by Ettorebigm** [saadeghi/daisyui#2037] [Ideas / feature requests]
Hello

The carousel is a charm, both the appearance and the simplicity of installment. But the flow of the images works only with a touch event, either on the phone or on a laptop trackpad, but not with the drag of a desktop mouse.

How to solve this issue ? Is there a workaround to achieve this behaviour ? 

Kind Regards

**Comment by saadeghi** [comment]
It's not an issue. 
As mentioned in the docs, carousel is just a scrollable area. It also provides scroll snapping.
And daisyUI as a CSS file, can not provide mouse dragging. You need JS for that.  
You can scroll it on a touch device by dragging, because that's how we scroll everything on touch devices. But on desktop, we can't scroll by dragging (unless we have some JS plugin for it)

Here are some JS plugins you can use to enable scroll by dragging using mouse:
- https://htmldom.dev/drag-to-scroll/
- https://ilyashubin.github.io/scrollbooster/
- https://vue-dragscroll.clebinfosys.com/
- https://www.npmjs.com/package/react-scroll-ondrag

There are more if you search on npm
